.github/CODEOWNERS

```diff
@@ -110,3 +110,36 @@
 
 /odoo/http.py @odoo/rd-security @odoo/rd-website
 /odoo/sql_db.py @odoo/rd-security
+
+# Logistics
+/addons/stock/ @odoo/rd-logistics
+/addons/stock_sms/ @odoo/rd-logistics
+/addons/stock_account/ @odoo/rd-logistics
+/addons/stock_picking_wave/ @odoo/rd-logistics
+/addons/stock_picking_batch/ @odoo/rd-logistics
+/addons/stock_dropshipping/ @odoo/rd-logistics
+/addons/stock_landed_costs/ @odoo/rd-logistics
+
+/addons/product_expiry/ @odoo/rd-logistics
+
+/addons/mrp/ @odoo/rd-logistics
+/addons/mrp_account/ @odoo/rd-logistics
+/addons/mrp_landed_costs/ @odoo/rd-logistics
+/addons/mrp_product_expiry/ @odoo/rd-logistics
+/addons/mrp_subcontracting/ @odoo/rd-logistics
+/addons/mrp_subcontracting_account/ @odoo/rd-logistics
+/addons/mrp_subcontracting_dropshipping/ @odoo/rd-logistics
+
+/addons/purchase/ @odoo/rd-logistics
+/addons/purchase_stock/ @odoo/rd-logistics
+/addons/purchase_mrp/ @odoo/rd-logistics
+/addons/purchase_requisition/ @odoo/rd-logistics
+/addons/purchase_requisition_stock/ @odoo/rd-logistics
+
+/addons/sale_mrp/ @odoo/rd-logistics
+/addons/sale_stock/ @odoo/rd-logistics
+/addons/sale_stock_margin/ @odoo/rd-logistics
+/addons/sale_purchase/ @odoo/rd-logistics
+/addons/sale_purchase_stock/ @odoo/rd-logistics
+
+/addons/repair/ @odoo/rd-logistics
```

Comment:
Hello,

Thank you for the update, explicit list looking good.

1. Don't add modules that don't exist in v12 here. Handle new modules progressively as forward ports go (and include the new one in master only when it is merged).
2. Please read and follow the instructions in the file, in particular those should be ordered alphabetically with the other modules, not added in a logistics category.

Reply:
Thanks, I updated my commit to:
- Respect the alphabetically order
- Remove the unexciting module (I will be add gradually with forward ports)